inlineplz/linters/__init__.py

```diff
@@ -641,6 +642,19 @@ def vendored_path(path):
         "autorun": True,
         "run_per_file": False,
     },
+    "tflint": {
+        "install": [
+            ["brew", "install", "tflint"],
+        ],
+        "help": ["tflint", "--help"],
+        "run": ["tflint", "--format=json"],
+        "rundefault": ["tflint", "--format=json"],
+        "dotfiles": [],
```

Comment:
we should support the tool's dotfiles - https://github.com/wata727/tflint#configuration